05-Jason-Luo/wk04/1-mon/calculator.rb

```diff
@@ -0,0 +1,99 @@
+def add_function
+    puts "What numbers to add?"
+    n1 = gets.chomp.to_f
+    n2 = gets.chomp.to_f
+    answer = n1+n2
+    puts "The sum is... #{answer}"
+end
+
+def divide_function
+    puts "What numbers to divide?"
+    n1 = gets.chomp.to_f
+    n2 = gets.chomp.to_f
+        answer = n1/n2 
+        puts "The answer is... #{answer}"
+end
```

Comment:
Line 10 & 13 are the only unique lines in each of these methods you've created.
To _DRY_ your code you could write a single, more generalized function like this:
```suggestion
def calculate (operation)
    puts "What numbers to #{ operation }?"
    n1 = gets.chomp.to_f
    n2 = gets.chomp.to_f
    answer = nil
    case operation
        when "divide"
            answer = n1/n2 
        when "add"
            answer = n1 + n2
        # etc . . .
    end
    
    puts "The answer is... #{ answer }"
end
```